udata/harvest/tests/test_base_backend.py

```diff
@@ -281,6 +287,78 @@ def test_harvest_datasets_get_deleted(self):
         for item in job.items:
             assert item.dataset is None
 
+    def test_no_datasets_duplication(self, app):
+        limit = app.config["HARVEST_AUTOARCHIVE_GRACE_DAYS"]
+        last_update = datetime.utcnow() - timedelta(days=limit - 1)
```

Comment:
I think bringing `HARVEST_AUTOARCHIVE_GRACE_DAYS` here makes it seem like it has some impact, but it should not right?

Reply:
Totally right! It was some leftover from a previous copy-paste, well spotted, thanks, i'll remove it straight away 🙏